assets/src/domain/eventEditor/ui/ticketAssignmentsManager/components/table/DateCell.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { parseISO } from 'date-fns';
+import classNames from 'classnames';
+
+import { RenderCellProps } from '../../types';
+import { getBackgroundColorClassName } from '@sharedEntities/datetimes/helpers';
+import { useTimeZoneTime } from '@appServices/hooks';
+import { blaBlaFormat } from '@appConstants/dateFnsFormats';
+
+const DateCell: React.FC<RenderCellProps> = ({ datetime }) => {
+	const bgClassName = getBackgroundColorClassName(datetime);
+	const { formatForSite: format } = useTimeZoneTime();
+	const stripeClassName = classNames('date-stripe', bgClassName);
+	const startDate = format(parseISO(datetime.startDate), blaBlaFormat);
```

Comment:
maybe reorder so that things are grouped together better

```suggestion
	const bgClassName = getBackgroundColorClassName(datetime);
	const stripeClassName = classNames('date-stripe', bgClassName);
	
	const { formatForSite: format } = useTimeZoneTime();
	const startDate = format(parseISO(datetime.startDate), blaBlaFormat);
```

Reply:
nice suggestion, thx